containers/layouts/ConversationGrouping.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { c } from 'ttag';
+import PropTypes from 'prop-types';
+import { Row, Label, Info, Field } from 'react-components';
+import ViewModeRadios from './ViewModeRadios';
+import StickyLabelsToggle from './StickyLabelsToggle';
+import { VIEW_MODE } from 'proton-shared/lib/constants';
+
+const { GROUP } = VIEW_MODE;
+
+const ConversationGrouping = ({
+    viewMode,
+    stickyLabels,
+    onChangeViewMode,
+    onToggleStickyLabels,
+    loadingViewMode,
+    loadingStickyLabels
+}) => {
+    const [show, set] = useState(viewMode === GROUP);
```

Comment:
I don't think you need to use a state to show or hide the `StickyLabels` row. Just use the `viewMode` you defined with `viewMode === GROUP`.

Reply:
You're totally right. I didn't realize that `ViewMode` works kind of as a state since it's reloaded every time the component re-renders. It's much better like this. Thanks a lot! :)